cmd/opera/launcher/db-transform.go

```diff
@@ -264,7 +264,7 @@ func transformComponent(datadir string, dbTypes, tmpDbTypes map[multidb.TypeName
 					return err
 				}
 				toMove[dbLocatorOf(e.New)] = true
-				newDB = batched.Wrap(newDB)
+				newDB = batched.Wrap(autocompact.Wrap(autocompact.Wrap(newDB, 1*opt.GiB), 16*opt.GiB))
```

Comment:
Can you explain why we always need to double wrap with difference `limit` like this?